integration-tests/oauth2/src/main/java/com/okta/spring/tests/oauth2/implicit/BasicImplicitFlowApplication.java

```diff
@@ -41,19 +55,60 @@ protected MethodSecurityExpressionHandler createExpressionHandler() {
         }
     }
 
-    @GetMapping("/")
-    @PreAuthorize("#oauth2.hasScope('email')")
-    public String getMessageOfTheDay(Principal principal) {
-        return "The message of the day is boring: " + principal.getName();
+    @Configuration
+    @Order(0)
+    static class ResourceSecurityConfigurer extends ResourceServerConfigurerAdapter {
+
+        @Override
+        public void configure(HttpSecurity http) throws Exception {
+            http.authorizeRequests()
+                    .antMatchers(HttpMethod.OPTIONS,"/**").permitAll()
+                    .anyRequest().authenticated();
+        }
     }
 
-    @GetMapping("/everyone")
-    @PreAuthorize("hasAuthority('Everyone')")
-    public String everyoneAccess(Principal principal) {
-        return "Everyone has Access: " + principal.getName();
+    @RestController
+    @CrossOrigin(origins = "http://localhost:8080")
+    public static class MessageOfTheDayController {
```

Comment:
These changes should be pulled out of this PR.

Reply:
It's actually required when updating the TCK version (otherwise I would agree)
We changed the tests after the initial version and I never pulled them back to this lib (the TCK was originally split out from this project)

Reply:
Gotcha. Lets pull the TCK updates out of this PR then 👍